lib/inspec/targets/dir.rb

```diff
@@ -11,13 +11,11 @@ module DirsHelper
     # TODO: remove `test` support for InSpec 1.0
     class ProfileDir
       def handles?(paths)
+        return true if paths.include?('inspec.yml')
```

Comment:
i.e. we don't need `controls` or `test` folder, just finding `inspec.yml` is enough.